README.md

```diff
@@ -162,9 +162,13 @@ This contains an Apache Airflow provider that providers the `@task.venv` decorat
 
 The `# syntax` line tells buildkit to user our Build frontend to process the Dockerfile into instructions.
 
-The example Dockerfile above gets converted into roughly following instructions
+The example Dockerfile above gets converted into roughly the following Dockerfile:
 
 ```Dockerfile
+# syntax=quay.io/astronomer/airflow-extensions:v1
```

Comment:
We don't need this here

Reply:
If we use this syntax we don't need to build the virtualenvs manually, like is shown in this example

Reply:
I'll try with @ashb 's suggestion below